packages/terra-list/tests/wdio/multi-select-list-spec.js

```diff
@@ -0,0 +1,207 @@
+
+    it('should highlight the third item upon clicking', () => {
+      browser.click('ul li:nth-child(3)');
+    });
+    Terra.should.matchScreenshot('first second and third item highlighted');
+
+    it('should unhighlight the second item upon clicking', () => {
+      browser.click('ul li:nth-child(2)');
+    });
+    Terra.should.matchScreenshot('first and third item highlighted');
+
+    it('should unhighlight the third item upon clicking', () => {
+      browser.click('ul li:nth-child(3)');
+    });
+    Terra.should.matchScreenshot('first item highlighted third item unhighlighted');
+
+    describe('Enter Key', () => {
+      before(() => browser.url('/#/raw/tests/terra-list/list/multi-select-list/default-multi-select-list'));
+
+      it('should select and highlight the first item upon enter', () => {
+        browser.keys(['Tab', 'Enter']);
+      });
+      Terra.should.matchScreenshot('first item highlighted');
+
+      it('should highlight the second item upon enter', () => {
+        browser.keys(['Tab', 'Enter']);
+      });
+      Terra.should.matchScreenshot('first and second item highlighted');
+
+      it('should highlight the third item upon enter', () => {
+        browser.keys(['Tab', 'Enter']);
+      });
+      Terra.should.matchScreenshot('first second and third item highlighted');
+
+      it('should unhighlight the second item upon enter', () => {
+        browser.keys(['Shift', 'Tab', 'Enter', 'Shift']);
+      });
+      Terra.should.matchScreenshot('first and third item highlighted');
+
+      it('should unhighlight the third item upon enter', () => {
+        browser.keys(['Tab', 'Enter']);
+      });
+      Terra.should.matchScreenshot('first item highlighted third item unhighlighted');
+    });
+
+    describe('Space Bar', () => {
+      before(() => browser.url('/#/raw/tests/terra-list/list/multi-select-list/default-multi-select-list'));
+
+      it('should highlight the selected items upon space keydown', () => {
+        browser.keys(['Tab', 'Space']);
+      });
+      Terra.should.matchScreenshot('first item highlighted');
+
+      it('should highlight the selected items upon space keydown', () => {
+        browser.keys(['Tab', 'Space']);
+      });
+      Terra.should.matchScreenshot('first and second item highlighted');
+
+      it('should highlight the selected items upon space keydown', () => {
+        browser.keys(['Tab', 'Space']);
+      });
+      Terra.should.matchScreenshot('first second and third item highlighted');
+
+      it('should highlight the selected items upon space keydown', () => {
+        browser.keys(['Shift', 'Tab', 'Space', 'Shift']);
+      });
+      Terra.should.matchScreenshot('first and third item highlighted');
+
+      it('should highlight the selected items upon space keydown', () => {
+        browser.keys(['Tab', 'Space']);
+      });
+      Terra.should.matchScreenshot('first item highlighted third item unhighlighted');
+    });
+    Terra.should.beAccessible();
```

Comment:
Looks like this guy is by itself? I think this is for the initial render. It might be more readable to push this guy up next to the initial should.matchScreenshot call